### ISSUE TITLE
jj should maybe exclude tags in the default revset

### ISSUE THREAD

**Issue by colemickens** [martinvonz/jj#681]
## Description

**tl;dr**:

> @martinvonz:
> ah. we should probably exclude tags in the default revset

**longer version**:

Thanks again @martinvonz for helping me get my head around some git+jj interactions. I do feel like I understand better what jj is doing and offers, as well as how to unstick myself in the future.

To recap, I was confused why I was seeing a lot of extra output in `jj log` from two separate clones that seemed to me, to be effectively the same.

Importantly:
1. `jj log` should only show commits that are not on a remote
2. `jj` seems to have imported all refs, including tags

This resulted in me having some tag refs that were pointing to commits that presumably don't exist in any of my remotes any longer (wouldn't be surprising for this repo on this machine).

For others, or myself in the future, here was the rough process:

```
/home/cole/code/nixpkgs/cmpkgs〉jj log | head -n20
@ 759906e25e0e dbe050c6771d cole.mickens@gmail.com 2022-10-27 10:16:17.000 -07:00
| (no description set)
o   3ca0c4bcc53b 910176db3b46 ilkecan@protonmail.com 2022-10-26 07:09:10.000 +00:00 cmpkgs   HEAD@git
:\  nushell: 0.69.1 -> 0.70.0
: ~
: o 99e776e6650b 47dc32666b5a viric@vicerveza.homeunix.net 2009-11-07 20:03:42.000 +00:00  backups/stdenv-updates@18281
: | Changing a bit the usage of 'platform' along the kernel, and expecting it to be
: o ed09bc1fec37 ae1d72088e2e viric@vicerveza.homeunix.net 2009-11-07 19:02:11.000 +00:00
: | Adding an arm kernel detail I forgot: ARCH=arm
[tons more output]
```

1. Why is "99e77"... showing up?
2. Well, is "backups/stdenv-updates" included in `jj log -r 'remote_branches()'` => no, well then it makes sense it appears.
3. Check git, `git for-each-ref refs/remotes/` and no, it's not there either, odd.

So we have a ref somewhere that jj picked up, but it's not from a remote.

Eventually I did a ripgrep:

```
/home/cole/code/nixpkgs/cmpkgs/.git〉rg backups
packed-refs
913:129a56627f08b76d3a1b3b15709023843fb25bdc refs/tags/backups/0.10-release@6725
914:7eec5e69f8699d6a4808ac77c434369445d62d37 refs/tags/backups/0.11-release@9315
915:9e3f729c35abe3999b414e1c2c859b1fd711ab75 refs/tags/backups/0.12-release@15293
916:f5c7d72813805ebfc73c738194fa05e02176c8f8 refs/tags/backups/0.5-release@989
917:b9686c27ed9ac993923b6e4ec2234dc0cf0b864d refs/tags/backups/0.5-stable@34171
```
